### ISSUE TITLE
Photo adjustment handle buttons have no accessible names for screen reader users

### ISSUE THREAD

**Issue by tamminivenkat-zz** [pikachu987/CropPickerView#7]
Accessibility not working as expected:

Photo adjustment handle buttons have no accessible names for screen reader users.
Give each different button a meaningful accessible name for screen reader users. 

![image-2020-10-21-20-57-44-987](https://user-images.githubusercontent.com/8720453/98644939-3ac44400-2357-11eb-866c-74c3145caa1a.png)

![image-2020-10-21-20-56-08-698](https://user-images.githubusercontent.com/8720453/98644920-33049f80-2357-11eb-802b-9197f3ab797c.png)

**Comment by pikachu987** [comment]
The access modifier of the internal buttons has been changed to 'pubilc'.

```swift
private let cropPickerView: CropPickerView = {
    let cropPickerView = CropPickerView()
    cropPickerView.backgroundColor = .black
    cropPickerView.leftTopButton.backgroundColor = .blue
    cropPickerView.topButton.backgroundColor = .green
    cropPickerView.rightTopButton.backgroundColor = .white
    cropPickerView.leftButton.backgroundColor = .yellow
    cropPickerView.rightButton.backgroundColor = .brown
    cropPickerView.leftBottomButton.backgroundColor = .cyan
    cropPickerView.bottomButton.backgroundColor = .magenta
    cropPickerView.rightBottomButton.backgroundColor = .orange
    cropPickerView.centerButton.backgroundColor = .purple
    return cropPickerView
}()
```

![Simulator Screen Shot - iPhone 12 Pro - 2021-03-27 at 22 11 04](https://user-images.githubusercontent.com/16880502/112721841-6e3fed80-8f49-11eb-8dad-01852a9916a0.png)
